1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,50 @@
-function setAlarm() {}
+function setAlarm() {
+  const input = document.getElementById("alarmSet"); //input id
+  let time = input.value;
+  const title = document.getElementById("timeRemaining"); //header id
+  title.innerText = `Time Remaining: ${timer(time)}`; //second move for tittle.innerText fixed, passes test 1 and 2
+  console.log(title.innerText);
+
+  const alarmVoice = document.createElement("audio");
+  const voiceSource = document.createElement("source");
+  voiceSource.setAttribute("src", "alarmsound.mp3");
+  alarmVoice.append(voiceSource);
+  title.append(alarmVoice);
+
+  let titleTimer = () => {
+    title.innerText = `Time Remaining: ${timer(time - 1)}`; //tittle.innerText is declared here first
+
+    if (time === 0) {
+      clearInterval(timeLoop);
+/*       setInterval(function () {
+        document.body.style.backgroundColor =
+          "#" + Math.floor(Math.random() * 16777215).toString(16);
+      }, 100); */
+      console.log("time is up");
+      playAlarm();
+    }
+    time--;
```

Comment:
would be good to add a condition here to prevent the timer from going to negative. 

Also, if I set the input to zero - it keeps going to negative numbers without ever stopping :)